.gitignore

```diff
@@ -3,7 +3,7 @@
 /test-report/
 /coverage/
 /build/
-*.tgz
+dreamit-graphql-server-5*.tgz
```

Comment:
This should be set back to `*.tgz` as we do not want to use tarballs within the project, this is just temporary to so we can work with the new graphql-testing and graphql-server-base major versions.

Reply:
reverted